lib/DXIL/DxilTypeSystem.cpp

```diff
@@ -633,7 +633,10 @@ void RemapObsoleteSemantic(DxilParameterAnnotation &paramInfo, DXIL::SigPointKin
       if (_strnicmp(semName.data(), "COLOR", 5) == 0) {
         RemapSemantic(semName, semFullName, "SV_Target", paramInfo, Context);
       }
-      else if (_strnicmp(semName.data(), "DEPTH", 5) == 0) {
+      if (_strnicmp(semName.data(), "VFACE", 5) == 0) {
+        RemapSemantic(semName, semFullName, "SV_IsFrontFace", paramInfo,
+                      Context);
+      } else if (_strnicmp(semName.data(), "DEPTH", 5) == 0) {
```

Comment:
The problem here is that this is in the outer conditional block `if (sigPoint == DXIL::SigPointKind::PSOut) {` consequently, it will only apply when the semantic is applied to a PS output, which it never will. It would need to be below similar to the special cases for POSITION and VPOS below, but with PSIn and size == 5.

Reply:
Ah .. I see. Let me just remove this change.